Documentation/ceph-nfs-crd.md

```diff
@@ -13,109 +14,96 @@ resource definition. This will spin up a cluster of
 
-#### For Ceph v16 or newer
-* `poolConfig`: (optional) The pool settings to use for the RADOS pool.
-  It matches the [CephBlockPool](ceph-block.md) specification.
-  The settings will be applied to a pool named `.nfs` on Ceph v16.2.7 or newer.
-
-#### For Ceph v15
-* `pool`: (mandatory) The Ceph pool where NFS configuration is stored.
-* `namespace`: (mandatory) The namespace in the `pool` where configuration objects will be stored.
-
-Rook ignores both `pool` and `namespace` ([see above](#for-ceph-v15)) settings when running Ceph v16
-or newer.
+* `active`: The number of active NFS servers. Rook supports creating more than one active NFS
+  server, but cannot guarantee high availability. For values greater than 1, see the
+  [known issue](#serveractive-count-greater-than-1) below.
+* `placement`: Kubernetes placement restrictions to apply to NFS server Pod(s). This is similar to
+  placement defined for daemons configured by the
+  [CephCluster CRD](https://github.com/rook/rook/blob/{{ branchName }}/deploy/examples/cluster.yaml).
+* `annotations`: Kubernetes annotations to apply to NFS server Pod(s)
+* `labels`: Kubernetes labels to apply to NFS server Pod(s)
+* `resources`: Kubernetes resource requests and limits to set on NFS server Pod(s)
+* `priorityClassName`: Set priority class name for the NFS server Pod(s)
+* `logLevel`: The log level that NFS-Ganesha servers should output.</br>
+  Default value: NIV_INFO</br>
+  Supported values: NIV_NULL | NIV_FATAL | NIV_MAJ | NIV_CRIT | NIV_WARN | NIV_EVENT | NIV_INFO | NIV_DEBUG | NIV_MID_DEBUG | NIV_FULL_DEBUG | NB_LOG_LEVEL
 
 
 ## Creating Exports
 When a CephNFS is first created, all NFS daemons within the CephNFS cluster will share a
 configuration with no exports defined.
 
-### For Ceph v16 or newer
-For Ceph v16.2.0 through v16.2.6, exports cannot be managed through the Ceph dashboard, and
-newly-created Ceph command line tools are lacking. We highly recommend using Ceph v16.2.7 or higher
-with NFS, which fixes bugs and streamlines export management, allowing exports to be created via the
-Ceph Dashboard and the Ceph CLI. With v16.2.7 or higher, the Ceph dashboard and Ceph CLI will be
-able to manage the same NFS exports interchangeably as desired.
-
-#### Using the Ceph Dashboard
+### Using the Ceph Dashboard
 Exports can be created via the
 [Ceph dashboard](https://docs.ceph.com/en/latest/mgr/dashboard/#nfs-ganesha-management) for Ceph v16
 as well. To enable and use the Ceph dashboard in Rook, see [here](ceph-dashboard.md).
 
-#### Using the Ceph CLI
+### Using the Ceph CLI
 The Ceph CLI can be used from the Rook toolbox pod to create and manage NFS exports. To do so, first
-ensure the necessary Ceph mgr modules are enabled and that the Ceph orchestrator backend is set to
-Rook.
+ensure the necessary Ceph mgr modules are enabled, if necessary, and that the Ceph orchestrator
+backend is set to Rook.
+
+#### **Enable the Ceph orchestrator if necessary**
```

Comment:
Does this header only apply to this paragraph through line 110? It has the appearance of also applying all the way to line 172.

Reply:
It doesn't. I wasn't sure how to easily link back to just this part from later in the doc. I'll play with it a bit more.